src/Manager/RuleManager.php

```diff
@@ -34,17 +47,11 @@ class RuleManager extends ManagerBase
     public function query($options = [])
     {
         $url = '/rules';
-        $url .= isset($options['visible_on_page']) ? "&visible_on_page={$options['visible_on_page']}" : '';
-        $url .= isset($options['prefetch']) ? "&prefetch={$options['prefetch']}" : true;
-        $url .= isset($options['sort']) ? "&sort={$options['sort']}" : 'asc';
-        $url .= isset($options['start']) ? "&start={$options['start']}" : 0;
-        $url .= isset($options['rows']) ? "&rows={$options['rows']}" : 10;
-        $url .= isset($options['sort_field']) ? "&sort_field={$options['sort_field']}" : 'updated';
-        $url .= isset($options['status']) ? "&status={$options['status']}" : 'published';
```

Comment:
I agree - but maybe we should call this out in the examples?  The biggest one is "prefetch" as it defaults to `false` in the API.